### PR TITLE
BL-877 DS Implement timestamping for create_many database method

### PR DESCRIPTION
**Pull request by stevenhastings**
## Description
By adding the timestamp to the create_many method we're are removing the theoretical wall that was standing in the way between it and our randomizer's ability to do its job. This sequence of events was causing some turbulence for Front-End if they ever decided to do something CRAZY like request more than a single document in one call. Next, I was having difficulty making an errorless connection to our database and realized that MongoClient requires TLS certificate for secure communication sometimes python is unable to request via TLS so we explicitly mention for MongoClient to request to MongoDB using certifi package. Certifi has been added to requirements.txt. If anyone has any questions or concerns, feel free to reach out and I'll do my best to field them. Happy Coding!!!

## Jira Ticket
[The Sandman the KanBan and Me](https://bloomtechlabs.atlassian.net/browse/BL-877)

## Type of Change
- [x] Bug fix

## Checklist
- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

## Loom Video
https://www.loom.com/share/1da01f72153f41779d5a5453c0dbc5c1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@ pydantic[email]==1.9.1
 vaderSentiment==3.3.2
 uvicorn[standard]==0.17.6
 gunicorn==20.1.0
+
```
Comment (BrokenShell) [inline requirements.txt:11]: This is good work, let's get rid of that blank line (11) in the requirements.txt file.

**Review by BrokenShell — CHANGES_REQUESTED** [review]
Good work, I have one very minor change for ya.

**Review by jack-semder — DISMISSED** [review]
Nice changes and a very detailed description of improvements, good work!

**Review by stevenhastings — COMMENTED** [review]
This resolved Roberts concern with the VOID! Also, the blank space in the requirements.txt file.

**Review by BrokenShell — APPROVED** [review]

**Comment by BrokenShell** [comment]
This looks great, good job!

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Nice catch! Seemingly small, but fixes issues down the line.

**Review by jinjahninjah — APPROVED** [review]

**Pull request merged**